src/MOLECULE/angle_cosine.cpp

```diff
@@ -72,6 +75,7 @@ void AngleCosine::compute(int eflag, int vflag)
     dely1 = x[i1][1] - x[i2][1];
     delz1 = x[i1][2] - x[i2][2];
 
+    domain->minimum_image(delx1, dely1, delz1);
```

Comment:
These should be removed, this is not how PBC's are handled in LAMMPS. Instead, it is done in the NTopo classes e.g. atom1 = domain->closest_image(i,atom1);